### PR TITLE
Update 2018-11-08-governance-and-project-organization.md

### PR DESCRIPTION
**Pull request by mgoelzer**

### CODE REVIEW

File: meeting-notes/2018-11-08-governance-and-project-organization.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Summary
 
-Call recording:  [https://youtu.be/YoTwZ5vFi80](https://youtu.be/YoTwZ5vFi80)
+Call recording:  [https://youtu.be/YoTwZ5vFi80](https://youtu.be/YoTwZ5vFi80) ([alternate copy](https://drive.google.com/open?id=1OFxpUSCxwHk0snepxF-BY9aSDFXAjEiB))
 
 We had a good but very wide-ranging discussion.  Key issues raised:
  - Do we even want "governance" at this time or is it too early?
```
Comment (daviddias) [inline meeting-notes/2018-11-08-governance-and-project-organization.md]: @mgoelzer no actual notes??

Comment (mgoelzer) [reply meeting-notes/2018-11-08-governance-and-project-organization.md]: @daviddias Alas, no. 😢But I don't think governance is something we're going to prioritize in the near term so whenever we revisit it I'll probably brush up by re-watching this video.

Comment (daviddias) [reply meeting-notes/2018-11-08-governance-and-project-organization.md]: I mean, no notes from the whole call?

Comment (mgoelzer) [reply meeting-notes/2018-11-08-governance-and-project-organization.md]: That's correct.

Comment (daviddias) [reply meeting-notes/2018-11-08-governance-and-project-organization.md]: No notes for me to review then #sadpanda 🐼 🙁

Comment (daviddias) [reply meeting-notes/2018-11-08-governance-and-project-organization.md]: ref: https://github.com/ipfs/team-mgmt/issues/493

Comment (mgoelzer) [reply meeting-notes/2018-11-08-governance-and-project-organization.md]: I'm sorry I made the panda so sad. 🐼

No disagreement with ipfs/team-mgmt#493; I would prefer notes too.  In practice, there's a lot of administrative work related to the libp2p project that falls through the cracks.  I have some ideas about why this happens and what we could do to remedy it, but I'm not sure this PR is the right place to get into that.

**Pull request merged**